ports/stm32/boards/Passport/tools/pubkey-to-c/pubkey-to-c

```diff
@@ -1,4 +1,7 @@
 #! /usr/bin/python
+# SPDX-FileCopyrightText: 2023 Foundation Devices, Inc. <hello@foundationdevices.com>
```

Comment:
As in #107, let's make sure the copyright year is the year of first creation.  Same for other files in this PR as well.